apps/mobile/src/features/settings/account-identifier/account-identifier-cell.tsx

```diff
@@ -19,7 +21,7 @@ export function AccountIdentifierCell({
   title,
 }: AccountIdentifierCellProps) {
   return (
-    <TouchableOpacity onPress={() => onUpdateAccountDisplayPreference(accountDisplayPref.type)}>
+    <Pressable onPress={() => onUpdateAccountDisplayPreference(accountDisplayPref.type)}>
```

Comment:
Should I be refactoring my work to use `Pressable` instead of `TouchableOpacity` ?

Reply:
cc @fabric-8 here bc not sure everywhere? In the settings design review we were asked to remove the opacity change on press of the 'cells'.

Reply:
Let's sync on this @fabric-8 and let me know if I need to update the homepage. I can gather some places where we use opacity to check with you